tm2/pkg/sdk/params/keeper.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/amino"
 	"github.com/gnolang/gno/tm2/pkg/sdk"
 	"github.com/gnolang/gno/tm2/pkg/store"
+	"github.com/golang/groupcache/lru"
```

Comment:
Maybe this lib is more maintained than the one used: https://github.com/hashicorp/golang-lru